python313Packages.lxmf: 0.6.3 -> 0.7.1, python313Packages.rns: 0.9.5 -> 0.9.6

fabaff opened the pull request:
Diff: https://github.com/markqvist/lxmf/compare/refs/tags/0.6.3...refs/tags/0.7.0

Changelog: https://github.com/markqvist/LXMF/releases/tag/0.7.0


<!--
^ Please summarise the changes you have done and explain why they are necessary here ^

For package updates please link to a changelog or describe changes, this helps your fellow maintainers discover breaking updates.
For new packages please briefly describe the package or provide a link to its homepage.
-->

## Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- Built on platform(s)
  - [ ] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [25.05 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) (or backporting [24.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) and [25.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2505.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

<!--
To help with the large amounts of pull requests, we would appreciate your
reviews of other pull requests, especially simple package updates. Just leave a
comment describing what you have tested in the relevant package/service.
Reviewing helps to reduce the average time-to-merge for everyone.
Thanks a lot if you do!

List of open PRs: https://github.com/NixOS/nixpkgs/pulls
Reviewing guidelines: https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#reviewing-contributions
-->

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 407106`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 8 packages built:</summary>
  <ul>
    <li>python312Packages.lxmf</li>
    <li>python312Packages.lxmf.dist</li>
    <li>python312Packages.nomadnet</li>
    <li>python312Packages.nomadnet.dist</li>
    <li>python313Packages.lxmf</li>
    <li>python313Packages.lxmf.dist</li>
    <li>python313Packages.nomadnet</li>
    <li>python313Packages.nomadnet.dist</li>
  </ul>
</details>

NickCao reviewed (dismissed):
License change.

fabaff committed:
python313Packages.lxmf: 0.6.3 -> 0.7.1

Diff: markqvist/LXMF@refs/tags/0.6.3...refs/tags/0.7.1

Changelog: https://github.com/markqvist/LXMF/releases/tag/0.7.1

fabaff committed:
python313Packages.rns: 0.9.5 -> 0.9.6

Diff: https://github.com/markqvist/Reticulum/compare/refs/tags/0.9.5...refs/tags/0.9.6

Changelog: https://github.com/markqvist/Reticulum/blob/0.9.6/Changelog.md

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 407106`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>python312Packages.lxmf</li>
    <li>python312Packages.lxmf.dist</li>
    <li>python312Packages.nomadnet</li>
    <li>python312Packages.nomadnet.dist</li>
    <li>rns (python312Packages.rns)</li>
    <li>rns.dist (python312Packages.rns.dist)</li>
    <li>python313Packages.lxmf</li>
    <li>python313Packages.lxmf.dist</li>
    <li>python313Packages.nomadnet</li>
    <li>python313Packages.nomadnet.dist</li>
    <li>python313Packages.rns</li>
    <li>python313Packages.rns.dist</li>
  </ul>
</details>

qbit commented:
ah, i forgot the license change in https://github.com/NixOS/nixpkgs/pull/407628. I'll close it

qbit commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 407106`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>python312Packages.lxmf</li>
    <li>python312Packages.lxmf.dist</li>
    <li>python312Packages.nomadnet</li>
    <li>python312Packages.nomadnet.dist</li>
    <li>rns (python312Packages.rns)</li>
    <li>rns.dist (python312Packages.rns.dist)</li>
    <li>python313Packages.lxmf</li>
    <li>python313Packages.lxmf.dist</li>
    <li>python313Packages.nomadnet</li>
    <li>python313Packages.nomadnet.dist</li>
    <li>python313Packages.rns</li>
    <li>python313Packages.rns.dist</li>
  </ul>
</details>

fabaff committed:
python313Packages.nomadnet: 0.6.2 -> 0.7.0

Diff: https://github.com/markqvist/NomadNet/compare/refs/tags/0.6.2...refs/tags/0.7.0

Changelog: https://github.com/markqvist/NomadNet/releases/tag/0.7.0

fabaff commented:
## `nixpkgs-review` result

Generated using [`nixpkgs-review`](https://github.com/Mic92/nixpkgs-review).

Command: `nixpkgs-review pr 407106`

---
### `x86_64-linux`
<details>
  <summary>:white_check_mark: 12 packages built:</summary>
  <ul>
    <li>python312Packages.lxmf</li>
    <li>python312Packages.lxmf.dist</li>
    <li>python312Packages.nomadnet</li>
    <li>python312Packages.nomadnet.dist</li>
    <li>rns (python312Packages.rns)</li>
    <li>rns.dist (python312Packages.rns.dist)</li>
    <li>python313Packages.lxmf</li>
    <li>python313Packages.lxmf.dist</li>
    <li>python313Packages.nomadnet</li>
    <li>python313Packages.nomadnet.dist</li>
    <li>python313Packages.rns</li>
    <li>python313Packages.rns.dist</li>
  </ul>
</details>
